### PR TITLE
For #11015 - Switch Default ETP Setting to Standard

### PR DESCRIPTION
**Pull request by ekager**
Screenshot_20200529-144621
Screenshot_20200529-144559

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by ekager** [comment]
Yeah waiting on a decision in #11015 about how we want to do this, and then will fix up all the tests!

**Comment by Amejia481** [comment]
👍

File: app/src/main/res/layout/onboarding_tracking_protection.xml [hunk 1]
```diff
@@ -71,7 +72,7 @@
         android:id="@+id/protection_standard_title"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="@string/onboarding_tracking_protection_standard_button"
+        android:text="@string/preference_enhanced_tracking_protection_standard_option"
```
Comment (liuche) [inline app/src/main/res/layout/onboarding_tracking_protection.xml:75]: For the Nightly PR, can we land a new onboarding string?

**Comment by liuche** [comment]
For a little context, we're doing this for the experiment that's running, but there will be a decision made for what we want to ship with for the Release release (so we should keep all the other strings around).

@ekager can also update screenshots when you get the chance? Standard (and not using the "recommended" string)

**Comment by ekager** [comment]
Screenshots updated and assuming UI tests pass this is ready to review! I didn't want to just ignore the UI tests but they're all based on strict, so I made them override the setting and test the strict content still 🤷‍♀️

**Pull request merged**